src/utils.js

```diff
@@ -13,12 +13,17 @@ const getCountries = () =>{
     return axios.get(`https://restcountries.eu/rest/v2/all`);
 }
 
-// check if the user's search result is in the list of countries
+// filter countries only for search text by name
 const searchByName = (countries, search) => {
-    const countryNames = countries.map(country => country.name);
-    const found = countryNames.find(name => name === search);
-    return (found ? `found ${search}!` : `No results for ${search} found :(`);
-
+    let newCountries = null;
+    if(search === "") {
+        newCountries = [...countries];
+        return newCountries;
```

Comment:
Why so many steps? Why spread & assign & return? Just do `return countries` it is clean&clear.